scripts/upload_to_sds.py

```diff
@@ -1,9 +1,46 @@
 #!/usr/bin/env python
 
```

Comment:
for usability, let's add a hint for users to run `python upload_to_sds.py --help` to see all options.